src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import './TodoInfo.scss';
+
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. It's better to handle the data preparation in a parent component and pass the necessary props to `TodoInfo`, rather than importing and using `UserInfo` directly within this component.